common/search/src/test/scala/weco/api/search/elasticsearch/ElasticsearchServiceTest.scala

```diff
@@ -101,13 +275,46 @@ class ElasticsearchServiceTest
         val multiSearchResponseFuture =
           elasticsearchService.executeMultiSearchRequest(multiSearchRequest)
 
-        whenReady(multiSearchResponseFuture) { multiSearchResponseEither =>
-          val multiSearchResponse = multiSearchResponseEither.right.value
-          val queryResults =
-            multiSearchResponse.items.map(_.response.right.value)
-          val returnedThings = queryResults.flatMap(_.to[ExampleThing])
+        whenReady(multiSearchResponseFuture) {
+          case (errors, searchResponses) =>
+            errors should have size (0)
```

Comment:
```suggestion
            errors should be empty
```

Reply:
`shouldBe empty` upset intellij syntax highlighting ... but it compiles fine.